packages/unison/unison.2.48.15v4/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "juergen@hoetzel.info"
+authors: [
+  "Jürgen Hötzel <juergen@hoetzel.info>"
+  "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+]
+homepage: "https://www.cis.upenn.edu/~bcpierce/unison/"
+bug-reports: "mailto:unison-users@yahoogroups.com "
+# dev-repo: "https://github.com/bcpierce00/unison"
+depends: [ "ocaml" {<= "4.09"} ]
+depopts: [ "lablgtk" ]
+patches: ["ocaml48.patch"]
```

Comment:
I don't understand why adding a patch when it's already a new version? Why not having the patch directly in the release? It would be less complicated.

Reply:
This patch is already included upstream, but not yet released.